crates/wash-cli/src/cmd/dev/session.rs

```diff
@@ -315,9 +320,13 @@ impl WashDevSession {
         };
 
         // Start the host in detached mode, w/ custom log file
+        let wasmcloud_version = wasmcloud_opts
+            .wasmcloud_version
+            .as_ref()
+            .map(|v| v.to_string())
```

Comment:
Isn't `.to_string()` unnecessary here since [`WasmcloudOpts` already requires it to be a string](https://github.com/wasmCloud/wasmCloud/pull/2527/files#diff-95950f0dc432da1e1d2cc34c0d34284d3d9a9cd8c8cf557705eb225a29f28296R162)?

Reply:
Oh, yes. I was thinking about adding the same functionality for dev too, but left it there as is. Creating a commit without the extra mapping. Thanks for the heads-up.